src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php

```diff
@@ -123,3 +123,25 @@ class="p-5"
 var attachmentMaxSize  = '<?php echo $this->uploadLimit; ?>';
 
 </script>
+<style type="text/css">
```

Comment:
@MeghaBiranje Add css in respective CSS class and use wrapper classes like tjBs5/tjBs3 or maybe extension wrapper if this is specific to some bootstrap version